--- ../src-base/minecraft/net/minecraft/block/BlockConcretePowder.java
+++ ../src-work/minecraft/net/minecraft/block/BlockConcretePowder.java
@@ -15,8 +15,9 @@
 import net.minecraft.util.math.BlockPos;
 import net.minecraft.world.IBlockAccess;
 import net.minecraft.world.World;
+import net.mite.block.BlockFallingEx;
 
-public class BlockConcretePowder extends BlockFalling
+public class BlockConcretePowder extends BlockFallingEx
 {
     public static final PropertyEnum<EnumDyeColor> field_192426_a = PropertyEnum.<EnumDyeColor>func_177709_a("color", EnumDyeColor.class);
 
@@ -27,7 +28,7 @@
         this.func_149647_a(CreativeTabs.field_78030_b);
     }
 
-    public void func_176502_a_(World p_176502_1_, BlockPos p_176502_2_, IBlockState p_176502_3_, IBlockState p_176502_4_)
+    public void onEndFalling(World p_176502_1_, BlockPos p_176502_2_, IBlockState p_176502_3_, IBlockState p_176502_4_)
     {
         if (p_176502_4_.func_185904_a().func_76224_d())
         {
