--- ../src-base/minecraft/net/minecraft/block/BlockDirt.java
+++ ../src-work/minecraft/net/minecraft/block/BlockDirt.java
@@ -15,8 +15,9 @@
 import net.minecraft.util.math.BlockPos;
 import net.minecraft.world.IBlockAccess;
 import net.minecraft.world.World;
+import net.mite.block.BlockFallingEx;
 
-public class BlockDirt extends Block
+public class BlockDirt extends BlockFallingEx
 {
     public static final PropertyEnum<BlockDirt.DirtType> field_176386_a = PropertyEnum.<BlockDirt.DirtType>func_177709_a("variant", BlockDirt.DirtType.class);
     public static final PropertyBool field_176385_b = PropertyBool.func_177716_a("snowy");
@@ -26,6 +27,7 @@
         super(Material.field_151578_c);
         this.func_180632_j(this.field_176227_L.func_177621_b().func_177226_a(field_176386_a, BlockDirt.DirtType.DIRT).func_177226_a(field_176385_b, Boolean.valueOf(false)));
         this.func_149647_a(CreativeTabs.field_78030_b);
+        this.fallingDistVec = 3;
     }
 
     public MapColor func_180659_g(IBlockState p_180659_1_, IBlockAccess p_180659_2_, BlockPos p_180659_3_)
