--- ../src-base/minecraft/net/minecraft/block/BlockPlanks.java
+++ ../src-work/minecraft/net/minecraft/block/BlockPlanks.java
@@ -12,8 +12,9 @@
 import net.minecraft.util.NonNullList;
 import net.minecraft.util.math.BlockPos;
 import net.minecraft.world.IBlockAccess;
+import net.mite.block.BlockFallingEx;
 
-public class BlockPlanks extends Block
+public class BlockPlanks extends BlockFallingEx
 {
     public static final PropertyEnum<BlockPlanks.EnumType> field_176383_a = PropertyEnum.<BlockPlanks.EnumType>func_177709_a("variant", BlockPlanks.EnumType.class);
 
@@ -22,6 +23,7 @@
         super(Material.field_151575_d);
         this.func_180632_j(this.field_176227_L.func_177621_b().func_177226_a(field_176383_a, BlockPlanks.EnumType.OAK));
         this.func_149647_a(CreativeTabs.field_78030_b);
+        this.fallingDistHor = 3;
     }
 
     public int func_180651_a(IBlockState p_180651_1_)
